third_party/glad/generate.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python
+# -*- coding: UTF-8 -*-
+
+import os
+import sys
+import argparse
+
+def generate(output):
+    dirname, _ = os.path.split(os.path.abspath(__file__))
+    command = 'cd ' + dirname + '/scripts;'
+    command += 'python -m glad --profile="compatibility" --api="gl=3.3" --generator="c" --spec="gl" --extensions="" --reproducible --quiet --out-path='
```

Comment:
Is this really necessary? Why can't we run the glad script directly from GN? (Or at the very least, use the module directly in the wrapper rather that sub-invoking python from python.)